### PR TITLE
add example tests to travis

### PR DESCRIPTION
**Pull request by lmcgupe**
Signed-off-by: guy perron <guy.perron@ericsson.com>

### CODE REVIEW

**Comment by lmcgupe** [comment]
ok it has worked now. had to increase the timeout on mocha-electron, seems it takes more time under ci.

File: config/scripts/travis-ci-build.sh [hunk 1]
```diff
@@ -6,4 +6,7 @@ npm install \
 && cd ../../examples/browser \
 && npm run bootstrap \
 && cd ../electron \
-&& npm run bootstrap
\ No newline at end of file
+&& npm run bootstrap \
+&& cd ../.. \
+&& npm run test-electron 
+
```
Comment (hexa00) [inline config/scripts/travis-ci-build.sh]: ?? this should not go back to the root directory as we discused ?

File: package.json [hunk 2]
```diff
@@ -105,6 +106,7 @@
     "watch": "tsc-watch --onSuccess \"npm run lint\"",
     "clean": "rimraf lib",
     "test": "mocha --opts ./config/mocha/mocha.opts src/**/*.spec.ts",
+    "test-electron": "electron-mocha --timeout 60000 --require ts-node/register examples/electron/**/*e-spec.ts",
```
Comment (hexa00) [inline package.json]: That's not merged in the test command anymore?

**Comment by lmcgupe** [comment]
it can't find test otherwise

it runs the test-electron which is in the root

________________________________

File: package.json (hunk 2 above)
Comment (lmcgupe) [inline package.json]: i split it back, works better

File: config/scripts/travis-ci-build.sh (hunk 1 above)
Comment (lmcgupe) [inline config/scripts/travis-ci-build.sh]: test-electron script is in the root, works well this way

**Comment by hexa00** [comment]
No it can't work like that, when we split out the examples from theia-core it won't work
It has to go in the examples dir, there's not reason for it not to work there

**Comment by lmcgupe** [comment]
i can move the test-electron script from root package.json to example package.json if that's what you refer to

**Comment by hexa00** [comment]
Yes that's what I mean, and it should be called "test" and merged with the other ones.. so there's only 1 command to run the tests

**Comment by lmcgupe** [comment]
the electron-mocha dependencies in the root package.json are no longer needed and should removed then

**Comment by hexa00** [comment]
You never added a electron-mocha dep in the root... afaik

**Comment by lmcgupe** [comment]
actually yes, this is where it was added all along until now...

**Comment by hexa00** [comment]
OK my bad, it's better now, one last remaining problem 
So now the tests are run for  test/*e-spec.ts"
And it so happens that the test is caleld test-example-spec.ts so the last e of example matches... 
That's not right I meant it as an expicit suffit so we shold change it to
test/**/.espec.ts
And change the file to .espec.ts

**Comment by lmcgupe** [comment]
done

**Comment by hexa00** [comment]
That LGTM for a first sanity test @akosyakov do you have comments?

**Pull request merged**